scripts/nn/examples/fm-regression-dummy-data.dml

```diff
@@ -0,0 +1,48 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# Imports
+source("staging/fm-regression.dml") as fm_regression
+
+# generate dummy data (just a sample!)
+n = 1000; d = 9; k=2;
+X = rand(rows=n, cols=d);
+y = rand(rows=n, cols=1);
+X_val = rand(rows=100, cols=d);
+y_val = rand(rows=100, cols=1);
+
+# Train
+[w0, W, V] = fm_regression::train(X, y, X_val, y_val);
+
+# Write model out
+write(w0, out_dir+"/w0");
+write(W, out_dir+"/W");
+write(V, out_dir+"/V");
```

Comment:
This file doesn't run due to the `out_dir` not being defined.  Since this is just a dummy script, I would just comment this out for now (and the accuracy write down below) like you did in the binary classification dummy script.